{"error": "ERREUR: 'Settings' object has no attribute 'SEW_SHELLS'\nTraceback (most recent call last):\n  File \"C:\\Users\\wiemb\\Desktop\\flask-ifc-clashdetection\\app\\routes.py\", line 80, in process_clash_detection\n    detector.add_model(path)\n  File \"C:\\Users\\wiemb\\Desktop\\flask-ifc-clashdetection\\app\\clash_detection.py\", line 109, in add_model\n    settings.set(settings.SEW_SHELLS, False)     # D\u00e9sactive la couture des coques\n  File \"C:\\Users\\wiemb\\Desktop\\flask-ifc-clashdetection\\venv\\lib\\site-packages\\ifcopenshell\\geom\\main.py\", line 226, in __getattr__\n    raise AttributeError(\"'Settings' object has no attribute '%s'\" % k)\nAttributeError: 'Settings' object has no attribute 'SEW_SHELLS'\n", "status": "failed"}